src/test/resources/org/apache/ibatis/migration/TemplateReaderTest.sql

```diff
@@ -0,0 +1,19 @@
+--
+--    Copyright 2010-2018 the original author or authors.
+--
+--    Licensed under the Apache License, Version 2.0 (the "License");
+--    you may not use this file except in compliance with the License.
+--    You may obtain a copy of the License at
+--
+--       http://www.apache.org/licenses/LICENSE-2.0
+--
+--    Unless required by applicable law or agreed to in writing, software
+--    distributed under the License is distributed on an "AS IS" BASIS,
+--    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+--    See the License for the specific language governing permissions and
+--    limitations under the License.
+--
+
+${line} * from beginning;
+select ${line} from middle;
+select * from ${line};
```

Comment:
We said we were going to test it at the beginning, middle, and end of the line.  This isn't at the end of the line; there's a semicolon after it.